R/SSplotMovementRates.R

```diff
@@ -116,7 +116,12 @@ SSplotMovementRates <-
     } # end subplot 1
 
     # subplot 2: time-varying movement
-    if (2 %in% subplots) {
+    if(is.null(MGparmAdj)) {
```

Comment:
Same comment as above about if{}else{}

Reply:
Also, for both of these lack of figure messages, should we make a note in the documentation for the functions that these figures are not available?